docs/warnings/create-element-types.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Invalid type for createElement
```

Comment:
Let’s call it `Invalid Element Type Warning` for consistency with other warning’s style. Same for filename, would be `invalid-element-type.md`.